src/datasources/asset/types/types.ts

```diff
@@ -29,6 +29,11 @@ export const AssetFeatureTogglesDefaults: AssetFeatureToggles = {
   assetSummary: true
 }
 
+export enum QueryReturnType {
```

Comment:
Name suggestion: AssetQueryReturnType to maintain consistency with AssetQueryType,